applications/crossbar/doc/ref/storage.md

```diff
@@ -24,18 +24,41 @@ schema for AWS attachment entry
 Key | Description | Type | Default | Required
 --- | ----------- | ---- | ------- | --------
 `handler` | What AWS service to use | `string('s3')` |   | `true`
-`name` | Friendly name for this configuration | `string()` |   | `false`
 `settings.bucket` | Bucket name to store data to | `string()` |   | `true`
+`settings.bucket_access_method` | how to access the host. | `string('auto' | 'host' | 'path')` |   | `false`
```

Comment:
shouldn't this be vhost as per erlcloud or the conversion is handled somewhere

Reply:
@tsromana nice finding. the change should be made in storage.attachment.aws.json and  `make apis` should fix the rest. want to make a pr for master and 4.1 after this gets merged ?

Reply:
sure why not :)